codecs/MWV.js

```diff
@@ -45,12 +45,16 @@ Field Number:
 
 var Codec = require('../lib/NMEA0183');
 
+Math.radians = function(degrees) {
+  return degrees * Math.PI / 180;
+};
```

Comment:
As there is no real need for monkey patching let's not do that.

Now that I look at this there's https://github.com/SignalK/signalk-parser-nmea0183/blob/master/lib/NMEA0183.js#L66

I'm not terribly fond of the big (and growing) if block, but to stay in line with the rest of the module I'd say conversions should go here.